base/sparse/spqr.jl

```diff
@@ -21,133 +21,293 @@ const ORDERING_BESTAMD = Int32(9) # try COLAMD and AMD; pick best#
 # tried.  If there is a high fill-in with AMD then try METIS(A'A) and take
```

Comment:
Well, actually it describes the default behavior so I guess it is okay to keep it. In theory, users could change the build options for SuiteSparse and in any case the comment is still correct, right?